<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>GuiItemAffects</class>
 <widget class="QDialog" name="GuiItemAffects" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="mp_gbAffects" >
     <property name="title" >
      <string>Affects</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="2" >
       <widget class="QToolButton" name="mp_tbAffect0" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QComboBox" name="mp_comboAffect4" />
      </item>
      <item row="3" column="0" >
       <widget class="QComboBox" name="mp_comboAffect3" />
      </item>
      <item row="2" column="0" >
       <widget class="QComboBox" name="mp_comboAffect2" />
      </item>
      <item row="1" column="0" >
       <widget class="QComboBox" name="mp_comboAffect1" />
      </item>
      <item row="0" column="0" >
       <widget class="QComboBox" name="mp_comboAffect0" >
        <property name="minimumSize" >
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="2" >
       <widget class="QToolButton" name="mp_tbAffect4" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QToolButton" name="mp_tbAffect3" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="mp_leAffect3" >
        <property name="minimumSize" >
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maxLength" >
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="mp_leAffect0" >
        <property name="minimumSize" >
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maxLength" >
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="mp_leAffect1" >
        <property name="minimumSize" >
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maxLength" >
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QToolButton" name="mp_tbAffect1" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QToolButton" name="mp_tbAffect2" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="mp_leAffect2" >
        <property name="minimumSize" >
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maxLength" >
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QLineEdit" name="mp_leAffect4" >
        <property name="minimumSize" >
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maxLength" >
         <number>20</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mp_pbOk" >
       <property name="text" >
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mp_pbSave" >
       <property name="text" >
        <string>Applica</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mp_pbCancel" >
       <property name="text" >
        <string>Annulla</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>mp_comboAffect0</tabstop>
  <tabstop>mp_leAffect0</tabstop>
  <tabstop>mp_tbAffect0</tabstop>
  <tabstop>mp_comboAffect1</tabstop>
  <tabstop>mp_leAffect1</tabstop>
  <tabstop>mp_tbAffect1</tabstop>
  <tabstop>mp_comboAffect2</tabstop>
  <tabstop>mp_leAffect2</tabstop>
  <tabstop>mp_tbAffect2</tabstop>
  <tabstop>mp_comboAffect3</tabstop>
  <tabstop>mp_leAffect3</tabstop>
  <tabstop>mp_tbAffect3</tabstop>
  <tabstop>mp_comboAffect4</tabstop>
  <tabstop>mp_leAffect4</tabstop>
  <tabstop>mp_tbAffect4</tabstop>
  <tabstop>mp_pbOk</tabstop>
  <tabstop>mp_pbSave</tabstop>
  <tabstop>mp_pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
